daprdocs/content/en/reference/cli/dapr-uninstall.md

```diff
@@ -43,6 +44,12 @@ You can also use option `--all` to remove .dapr directory, Redis, Placement and
 dapr uninstall --all
 ```
 
+You can specify a different container runtime while setting up Dapr. If you omit the `--container-runtime` flag, the default container runtime is Docker.
```

Comment:
What does it mean to uninstall this. This is not clear, is this to replace this with an alternative runtime? Can you explain the scenario more here?

```suggestion
You can specify a different container runtime uninstalling Dapr. If you omit the `--container-runtime` flag, the default container runtime is Docker.
```

Reply:
@pravinpushkar - Re this. https://github.com/dapr/cli/pull/1041#issuecomment-1255004312 Does this work on LInux?

Reply:
@msfussell Podman can be installed on all platforms- https://podman.io/getting-started/installation.html
So, I think there should not be any issue with dapr using Podman. Currently our GH workflow for macos uses Podman.

Reply:
@msfussell Added a separate page for podman just like https://docs.dapr.io/operations/hosting/self-hosted/self-hosted-with-docker/. Podman commands are similar to Docker and dapr run should behave in the same manner. I have the content for what I was able to verify.